deploy/runtime/ui/PV Losses.json

```diff
@@ -1981,27 +2059,27 @@
                 },
                 "X": {
                     "Type": 3.0,
-                    "Integer": 21.0
+                    "Integer": 315.0
                 },
                 "Y": {
                     "Type": 3.0,
-                    "Integer": 591.0
+                    "Integer": 663.0
                 },
                 "Width": {
                     "Type": 3.0,
-                    "Integer": 813.0
+                    "Integer": 516.0
                 },
                 "Height": {
                     "Type": 3.0,
-                    "Integer": 24.0
+                    "Integer": 108.0
                 },
                 "Tool Tip": {
                     "Type": 5.0,
                     "String": ""
                 },
                 "Caption": {
                     "Type": 5.0,
-                    "String": "AC losses apply to the electrical output of the inverter and account for losses not calculated by the inverter performance model."
+                    "String": "The PV Subhourly Clipping Correction calculates expected losses from inverter clipping with the Allen method. These inverter clipping losses are calculated at each timestep by indexing a matrix of DNI clearness index and Clipping Potential. The losses are only applicable for hourly timeseries weather data. The suggested matrix to use in this calculation is in the PV Subhourly Clipping dropdown below. This matrix should only be changed if you have alternative matrix values calculated for a particular site. "
```

Comment:
This comment needs to be updated now that the matrix isn't in the UI anymore.

Reply:
Updating comment. @cpaulgilman may need to wordsmith